app/models/concerns/account/avatar.rb

```diff
@@ -4,7 +4,7 @@ module Account::Avatar
   extend ActiveSupport::Concern
 
   IMAGE_MIME_TYPES = ['image/jpeg', 'image/png', 'image/gif', 'image/webp'].freeze
-  LIMIT = 2.megabytes
+  LIMIT = 10.megabytes
```

Comment:
It might make sense to use the same logic as for the Preview Cards:
```suggestion
  LIMIT = Rails.configuration.x.use_vips ? 8.megabytes : 2.megabytes
```